config/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+hello.preamble=application properties
+hello.epilogue=application properties
```

Comment:
Are these values used? Also, all fields hold the same value.

Reply:
The point is to be sure they are not used -> they are overridden by other config sources (OC ones). Also setting defaults means existing tests are not affected by these changes.

Reply:
Ok, then replace the content to something like "preamble-never-used-value" so it could be easily understood just by looking at the file

Reply:
sure, I'll do it.

Reply:
@fedinskiy any particular reason for dashes? I'll need to unify the values of all the properties (in secrets/configmaps) so I wonder if you want dashes everywhere or just here?

Reply:
Done.